patroni/ha.py

```diff
@@ -257,9 +271,15 @@ def manual_failover_process_no_leader(self):
         return self._is_healthiest_node(members, check_replication_lag=False)
 
     def is_healthiest_node(self):
+        if self.is_paused() and self.cluster.failover and not self.cluster.failover.scheduled_at:
```

Comment:
if self.cluster.failover and not ( self.cluster.failover.scheduled_at and self.is_paused()) covers both line 274 and line 286-287. I think line 274 should be moved below line 284, we might not want to initiate manual failover on nofailover nodes or on the nodes running the leader.